### PR TITLE
Update standard transformer protos to include valueType in features

### PR DESCRIPTION
**Pull request by ariefrahmansyah**
…ures

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] Added unit test, integration, and/or e2e tests
- [ ] Tested locally
- [ ] Updated documentation
- [ ] Update Swagger spec if the PR introduce API changes
- [ ] Regenerated Golang and Python client if the PR introduce API changes

### CODE REVIEW

File: api/cmd/transformer/.env.sample [hunk 1]
```diff
@@ -1,6 +1,6 @@
 LOG_LEVEL=DEBUG
 
-STANDARD_TRANSFORMER_CONFIG={"transformerConfig":{"feast":[{"project":"default","entities":[{"name":"driver_id","valueType":"STRING","jsonPath":"$$.driver_id"}],"features":[{"name":"driver_statistics:avg_daily_trips_3","defaultValue":0},{"name":"driver_statistics:conv_rate","defaultValue":0}]}]}}
+STANDARD_TRANSFORMER_CONFIG={"transformerConfig":{"feast":[{"project":"default","entities":[{"name":"driver_id","valueType":"STRING","jsonPath":"driver_id"}],"features":[{"name":"driver_statistics:avg_daily_trips_3","valueType":"STRING","defaultValue":"0"},{"name":"driver_statistics:conv_rate","defaultValue":"0"}]}]}}
```
Comment (pradithya) [inline api/cmd/transformer/.env.sample:3]: Can you rebase this part

Comment (ariefrahmansyah) [reply api/cmd/transformer/.env.sample]: Rebased and pushed

Comment (pradithya) [reply api/cmd/transformer/.env.sample]: Are you sure? it still remove `$$.` in jsonPath

Comment (ariefrahmansyah) [reply api/cmd/transformer/.env.sample]: The json parser library that we used doesn't need `$.` prefix

Comment (ariefrahmansyah) [reply api/cmd/transformer/.env.sample]: Ah, I didn't realize that you've changed `https://github.com/buger/jsonparser` to: `github.com/oliveagle/jsonpath`

Brb, updating the PR

**Review by pradithya — APPROVED** [review]
LGTM

**Pull request merged**